pathmind/simulation.py

```diff
@@ -114,13 +114,20 @@ def run(
         Runs a simulation with a given policy. In Reinforcement Learning terms this creates a
         "rollout" of the policy over the specified number of episodes to run in the simulation.
 
-        :param policy: A Pathmind Policy (local, server, or random)
+        :param policy: A Pathmind Policy (local, server, or random). Default is random.
         :param out_csv: If you specify an output CSV file, complete results of the first episode will be stored there.
         :param summary_csv: If you specify a summary CSV file, a summary of reward terms over all episodes will be
             stored in that file.
         :param num_episodes: the number of episodes to run rollouts for.
         :param sleep: Optionally sleep for "sleep" seconds to make debugging easier.
         """
+
+        if not policy:
+            from pathmind.policy import Random
```

Comment:
I wonder how you get around the circular import problem incurred by this. I'm rel. sure `policy` imports `Simulation`. If it wasn't for that, I would have tackled this PR a while ago.

We could pack everything into `simulation.py`.

Reply:
@maxpumperla - tbh, I don't get why this works this way. 😁 I agree it's weird so I'm okay closing this PR.

If we were to do refactoring, I might go the other way and move more to the file(s) not simulation.py. Then simulation.py could focus on being a skeleton list of functions that a user will have to fill in to have a working env. What do you think? Is this just making things too complicated?

Reply:
ha, the problem I see is that `train` etc. are methods on the interface. So we can't really factor that out. Let's me sit on this for a bit :D

Reply:
@slinlee your import at runtime makes this work... moving up the import would break the test. I'm ok with merging this, but maybe we should put a "don't move import statement warning" there or so.

Reply:
@maxpumperla - good idea, I just added the note about the import statement.